### PR TITLE
Stats: UI Theme Tracking

### PR DESCRIPTION
**Pull request by gblakeman**
Tracks the current UI theme at plugin open:

Watches for tracking events related to opening Realish’s UI (“show-plugin”) and inserts a param (`interfaceTheme`) onto `eventData` set to either “light” or “dark” based on the current theme.

### CODE REVIEW

**Comment by gblakeman** [comment]
Converting back to draft to implement as a single event at startup.

File: src/GUI.ts [hunk 1]
```diff
@@ -48,7 +48,23 @@ const app = new App({
...
+const track = (eventPayload: {
+  eventName: string,
+  eventData: any,
+  eventType: string,
+  userId: string,
+}) => {
+  const { eventData, eventName } = eventPayload;
+
+  // add theme data on plugin UI open
+  if (eventName === 'show-plugin') {
```
Comment (joeyyang) [inline src/GUI.ts:60]: we don't have access to `document` from `code.ts` right? if we did, i'd suggest potentially attaching `interfaceTheme` to `eventData` there but if not this makes sense to me!

**Review by joeyyang — APPROVED** [review]
Small question about where we query for dark mode but this looks good!

**Review by lynnerang — APPROVED** [review]
👍

**Pull request merged**